### PR TITLE
add tcp read chunk size option

### PR DESCRIPTION
**Pull request by siddontang**
PTAL @BusyJay @overvenus

### CODE REVIEW

File: src/channel.rs [hunk 1]
```diff
@@ -41,6 +41,9 @@ const OPT_SO_REUSE_PORT: &'static [u8] = b"grpc.so_reuseport\0";
 const OPT_SSL_TARGET_NAME_OVERRIDE: &'static [u8] = b"grpc.ssl_target_name_override\0";
 const OPT_STREAM_INITIAL_WINDOW_SIZE: &'static [u8] = b"grpc.http2.lookahead_bytes\0";
 const PRIMARY_USER_AGENT_STRING: &'static [u8] = b"grpc.primary_user_agent\0";
+const OPT_TCP_READ_CHUNK_SIZE: &'static [u8] = b"grpc.experimental.tcp_read_chunk_size\0";
```
Comment (BusyJay) [inline src/channel.rs]: We should keep option strings together.

File: src/channel.rs [hunk 2]
```diff
@@ -212,9 +231,9 @@ impl ChannelBuilder {
         Channel {
             cq: self.env.pick_cq(),
             inner: Arc::new(ChannelInner {
-                                _env: self.env,
-                                channel: channel,
-                            }),
+                _env: self.env,
```
Comment (overvenus) [inline src/channel.rs]: Avoid unrelated change.

Comment (siddontang) [inline src/channel.rs]: rustfmt formats it automatically, I think it is fine here.

**Comment by siddontang** [comment]
PTAL @BusyJay @overvenus
